TESTING.md

```diff
@@ -0,0 +1,138 @@
+# Tardigrade-ci Testing Documentation
+
+Testing tardigrade-ci implementation with various setup on local environment.
+
+## Testing
+
+This project can be utilized one of two ways, via docker or via a Makefile include.
+
+### Test Setup
+
+Configure or run the following script to setup a test project environment:
+
+```bash
+#!/bin/bash
+
+TESTDIR=$(mktemp --tmpdir -d tardigrade-ci.XXXXXXXXXX)
+cd $TESTDIR
+git init
+echo "FROM plus3it/tardigrade-ci:latest" > Dockerfile
+echo "SHELL := /bin/bash\n" > Makefile
```

Comment:
Running through this, and noticed this one issue, the `\n` here causes an error:

```suggestion
echo "SHELL := /bin/bash" > Makefile
```